.circleci/config.yml

```diff
@@ -157,28 +179,75 @@ workflows:
     jobs:
       - trigger-workflows
 
+  # Trigger tag build when tag and not already building a specific package workflow
+  tag:
+    when: 
+      and:
+        - << pipeline.git.tag >>
+        - not: << pipeline.parameters.wpe-headless-plugin >>
+    jobs:
+      - trigger-tag:
+          filters:
+            tags:
+              only: /^plugin\/wpe-headless\/\S+/
+
   # Workflows defined for each package.
   plugin-wpe-headless:
     when: << pipeline.parameters.wpe-headless-plugin >>
     jobs:
-      - checkout
+      - checkout:
```

Comment:
These are all added, cause by default circleci doesn't trigger on tag event, so when we want to, we have to add this to all jobs in the workflow that we want responding to the tag.

Reply:
Interesting! Might be worth inline-comomenting this particular tidbit.